src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/26.diff.sql

```diff
@@ -0,0 +1,478 @@
+﻿/*************************************************************
+    This migration passes in a new parameter to the upsert
+    stored procedure. The parameter tracks whether or not a
+    version check is required on an update, which ultimately
+    enables PUT creates when the versioning policy is set to
+    "versioned-update".
+**************************************************************/
```

Comment:
Modifications made to the `UpsertResource_6` stored procedure are easier to review in `UpsertResource_6.sql`.